docker/Dockerfile.debug

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:xenial
+RUN apt-get update && \
+    apt-get install -y \
+    curl \
+    iptables \
+    iproute2 \
+    iputils-ping \
+    dnsutils \
+    netcat \
+    tcpdump \
+    net-tools \
+    libc6-dbg gdb valgrind \
+    vim \
+    nano \
+    emacs \
+    wrk
+ADD proxy-redirection-clear /usr/local/bin/proxy-redirection-clear
+ADD proxy-redirection-restore /usr/local/bin/proxy-redirection-restore
```

Comment:
Yes, but bazel violates layering of docker images. it doesn't do good job caching layers on top of pulled images.